filter/etc/e-smith/templates/etc/httpd/admin-conf.d/rspamd.conf/10base

```diff
@@ -13,13 +14,25 @@
     }
 
     my $trustedNetworks = join(" ", NethServer::TrustedNetworks::list_cidr());
+    my $access = ((${'httpd-admin'}{'access'}|| '') =~ 'red')? "all granted":"ip $trustedNetworks";
```

Comment:
If we go with PAM auth (like httpd-admin) we can remove the IP access policy completely

Reply:
ok